assembly_and_binning/tutorial_assembly_and_binning.md

```diff
@@ -0,0 +1,634 @@
+
+**Challenge 1**
+
+Look inside the contaminants folder. What contaminant was removed by atlas?
+
+<details>
+  <summary>Challenge Answer</summary>
+
+To look inside the contaminants folder using the following command:
+```
+$ ls SRR8859675/sequence_quality_control/contaminants/
+```
+You'll see:
+```
+PhiX_R1.fastq.gz  PhiX_R2.fastq.gz  PhiX_se.fastq.gz
+```
+PhiX is the contaminant sequence that is removed by ATLAS by default
+</details>
+
+
+
+#### Rerunning atlas steps
+
+Note that you can rerun `atlas run qc` and (as long as you succeeded the first time) atlas won't rerun anything! 
+That's because atlas (and the snakemake tool its built on) tracks what's been done and won't rerun anything that has suceeded!
+
+If you want to rerun the QC step, you can rename (or remove) the output of the atlas step, and rerun `atlas run qc`.
+
+### Assembly
+
+QC. Done. Yay! 
+Now that our reads are cleaned up and we have removed contaminants, let's assemble. 
+Again, let's do a dry run first to see what steps atlas will run. 
+
+```
+atlas run assembly -n
+```
+
+```
+Would remove temporary output finished_QC
+Job stats:
+job                             count    min threads    max threads
+----------------------------  -------  -------------  -------------
+align_reads_to_final_contigs        1              8              8
+assembly                            1              1              1
+assembly_one_sample                 1              1              1
+bam_2_sam_contigs                   1              4              4
+build_assembly_report               1              1              1
+calculate_contigs_stats             2              1              1
+combine_contig_stats                1              1              1
+convert_sam_to_bam                  1              4              4
+do_not_filter_contigs               1              1              1
+error_correction                    1              8              8
+finalize_contigs                    1              1              1
+get_contigs_from_gene_names         1              1              1
+init_pre_assembly_processing        1              1              1
+merge_pairs                         1              8              8
+pileup                              1              8              8
+predict_genes                       1              1              1
+rename_contigs                      1              4              4
+rename_megahit_output               1              1              1
+run_megahit                         1              8              8
+total                              20              1              8
+```
+
+Lots of things! Let's start the assembly and then we'll talk through these steps (takes ~8 min).
+
+```
+atlas run assembly
+```
+
+Megahit is the meat and potatoes of this pipeline. 
+It orchestrates the assembly of our reads, generating long(er) contigs. 
+(We ran it during the initial assembly tutorial, too.)
```

Comment:
?? isn't this the first assembly tutorial? did I miss one?

Reply:
nope, you missed nothing, good catch! That was a relic comment from 2019.
```suggestion
```